pkg/controllers/clusterresourceplacement/placement_status.go

```diff
@@ -142,6 +142,9 @@ func (r *Reconciler) setResourceConditions(ctx context.Context, crp *fleetv1beta
 		if unselected[i].ClusterScore != nil {
 			scheduledCondition.Message = fmt.Sprintf(resourcePlacementConditionScheduleFailedWithScoreMessageFormat, unselected[i].ClusterName, unselected[i].ClusterScore, unselected[i].Reason)
 		}
+		if unselected[i].Reason == fmt.Sprintf("specified cluster \"%s\" is not found", unselected[i].ClusterName) {
```

Comment:
just copy the reason should be enough

Reply:
I would suggest to copy reason directly from the cluster decision. It means we will build the whole message in the scheduler instead.